tools/ci_build/github/linux/docker/Dockerfile.manylinux2014_rocm5.3

```diff
@@ -18,7 +18,17 @@ LABEL maintainer="The ManyLinux project"
 # of the base docker image. Remove this line moving forward.
 RUN yum install -y hipify-clang
 
-RUN yum remove -y devtoolset\* git\* && conda remove -y cmake
+# CMake
+ENV CMAKE_VERSION=3.24.2
+RUN cd /usr/local && \
+    wget -q -O - https://github.com/Kitware/CMake/releases/download/v${CMAKE_VERSION}/cmake-${CMAKE_VERSION}-Linux-x86_64.tar.gz | tar zxf -
+ENV PATH=/usr/local/cmake-${CMAKE_VERSION}-linux-x86_64/bin:${PATH}
+
+# rocm lib
+RUN yum install -y libpython3.8-dev python3-pip miopen-hip-devel rocblas-devel && \
```

Comment:
Why do you install libpython3.8-dev? Manylinux docker images should not have this.

Reply:
And where does this package from? Does Centos ever have this package?

Reply:
And why do you install "python3-pip"?

Reply:
Remove them.